README.md

```diff
@@ -0,0 +1,45 @@
+# uome2
+
+A brief description of what this project does and who it's for
+
+## Tech Stack
+
+**Client:** Flutter, bloc,
+
+**Server:** Node, Express
+
+## Sample of Clean Architecture
+
+![App Screenshot](https://user-images.githubusercontent.com/7877772/45727888-97f15e80-bc00-11e8-888a-24c190959a8d.jpg)
+
+## Related
+
+Here are of Clean Architecture with Nodejs, TypeScript and Mongo.
+
+[orginal blog post](https://dev.to/japhernandez/clean-architecture-with-nodejs-typescript-and-mongo-3ene)
+
+## project Structure paths
+
+#### Domain
+
+Inside Model Folder We create the entity with the corresponding attributes like user...
```

Comment:
Instead of explaining the layers and the folders in them, we can explain them as if they are components.